scripts/CEE/retrieve-pdbs/script.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+set -e
+set -o nounset
+set -o pipefail
+
+# Function to retrieve all namespaces starting with openshift
+get_openshift_namespaces() {
+    oc get namespaces -o json | jq -r '.items[].metadata.name' | grep '^openshift'
+}
+
+# Function to retrieve and display PDBs for a given namespace
+get_and_display_pdbs() {
+    local namespace="$1"
+    echo "---------------------------------"
+    echo "PDBs in namespace: $namespace"
+    echo ""
+    oc get poddisruptionbudget -n "$namespace"
```

Comment:
I have some concerns as it gets data from customer namespaces.  I would like to understand more on the purpose on getting the PDB. 
If that's for health check, I think it is better to only output relevant data, eg, only output "It has bad PDBs in customer namespace"/"All PDBs look good in customer namespace".   Then guide customer to check the PDBs with certain commands.

Reply:
Hi @feichashao 

Let me share more context behind the idea of the initial script development.

- In proactive cases, the MCS team is interested in knowing if there are restrictive PDBs that could affect the nodes roll-out
- By default, we only read openshift-* to avoid reading data from the custiomer namespaces
- Added --all or specific namespaces option to debug certain cases where customers have pod rotation or workload migration issues, simply to verify these configurations without requesting a namespace inspection. (example, new machinePool that requires migrating the entire workload)


For MCS team, the --restrictive would be enough.
The rest are additions that should be able to be dispensed with or open an OHSS for troubleshooting.